scripts/tests/systemd/firezone-client-headless.service

```diff
@@ -35,6 +35,7 @@ SystemCallFilter=@aio @basic-io @file-system @io-event @network-io @signal @syst
 UMask=077
 
 Environment="FIREZONE_API_URL=ws://localhost:8081"
+# TODO: Remove after #6163 gets into a release
```

Comment:
Is there a way we can run the compat tests and integration tests with different parameters or something? Leaving this `FIREZONE_DNS_CONTROL` var in place means that we're missing out on test coverage for the current version

Reply:
> The question is which one?

I think checking out an old revision is going to give us more headaches because it means we can't introduce new things. The point of the compat tests is to ensure we only make backwards-compatible changes.

You can temporarily remove it and see if the integration tests pass. That will give you at least a once-off coverage.